Sources/ArcGISToolkit/Components/Popups/PopupView.swift

```diff
@@ -95,21 +95,19 @@ public struct PopupView: View {
         let popupElements: [PopupElement]
         
         var body: some View {
-            ScrollView {
-                VStack(alignment: .leading) {
-                    ForEach(popupElements) { popupElement in
-                        switch popupElement {
-                        case let popupElement as AttachmentsPopupElement:
-                            AttachmentsPopupElementView(popupElement: popupElement)
-                        case let popupElement as FieldsPopupElement:
-                            FieldsPopupElementView(popupElement: popupElement)
-                        case let popupElement as MediaPopupElement:
-                            MediaPopupElementView(popupElement: popupElement)
-                        case let popupElement as TextPopupElement:
-                            TextPopupElementView(popupElement: popupElement)
-                        default:
-                            EmptyView()
-                        }
+            List {
+                ForEach(popupElements) { popupElement in
+                    switch popupElement {
+                    case let popupElement as AttachmentsPopupElement:
+                        AttachmentsPopupElementView(popupElement: popupElement)
+                    case let popupElement as FieldsPopupElement:
+                        FieldsPopupElementView(popupElement: popupElement)
+                    case let popupElement as MediaPopupElement:
+                        MediaPopupElementView(popupElement: popupElement)
+                    case let popupElement as TextPopupElement:
+                        TextPopupElementView(popupElement: popupElement)
+                    default:
+                        EmptyView()
                     }
                 }
             }
```

Comment:
`plain` isn't great because of all the extra separators:

![image](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/2257493/9b8fe86f-fdf6-4877-a8c4-d0f7c9f7737e)

It might look better if the view was embedded in a navigation view. I'll try that.